### PR TITLE
[FLINK-12360][chinese-translation]Translate Jobs and Scheduling Page …

### PR DESCRIPTION
**Pull request by Armstrongya**
Translate the internal page "https://ci.apache.org/projects/flink/flink-docs-master/internals/job_scheduling.html" to Chinese 

## What is the purpose of the change
This pull request completes the Chinese translation of "job_scheduling" from official documents.

## Brief change log
 - *translates internal document "job_scheduling" into Chinese*

## Verifying this change

- *This change is to add a new translated document.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: docs/internals/job_scheduling.zh.md [hunk 1]
```diff
@@ -22,81 +22,52 @@ specific language governing permissions and limitations
...
-In case that the user cancels the job, it will go into the *cancelling* state.
-This also entails the cancellation of all currently running tasks.
-Once all running tasks have reached a final state, the job transitions to the state *cancelled*.
+如果用户取消了 job 话，它会进入到 *cancelling* 状态，并取消所有正在运行的 task。当所有正在运行的 task 进入到最终状态的时候，job 进入 *cancelled* 状态。
 
-Unlike the states *finished*, *canceled* and *failed* which denote a globally terminal state and, thus, trigger the clean up of the job, the *suspended* state is only locally terminal.
-Locally terminal means that the execution of the job has been terminated on the respective JobManager but another JobManager of the Flink cluster can retrieve the job from the persistent HA store and restart it.
-Consequently, a job which reaches the *suspended* state won't be completely cleaned up.
+跟 *finished* 不同的是，*canceled* 和 *failed* 会导致全局的终结状态，并且触发作业的清理，*suspended* 状态只是一个局部的终结。
+局部的终结意味着作业有一部分 JobManager 被终结，但是另外一部分 JobManager 依然可以从高可用存储里获取作业并重启。因此一个处于 *suspended* 状态的作业不会被彻底清理掉。
```
Comment (wuchong) [inline docs/internals/job_scheduling.zh.md]:
```suggestion
局部的终结意味着作业的执行已经被对应的 JobManager 终结，但是集群中另外的 JobManager 依然可以从高可用存储里获取作业信息并重启。因此一个处于 *suspended* 状态的作业不会被彻底清理掉。
```

**Comment by wuchong** [comment]
@flinkbot   approve all

File: docs/internals/job_scheduling.zh.md (hunk 1 above)
Comment (Armstrongya) [reply docs/internals/job_scheduling.zh.md]: @wuchong Thanks for your refined translation. It's more reasonable. Does this means my first pull request will merge to Flink codebase ? 
ps: How to move my jira task issue (https://issues.apache.org/jira/browse/FLINK-12360) to subtask issue of "Support Chinese Documents for Apache Flink"  (https://issues.apache.org/jira/browse/FLINK-11529) ?

Comment (wuchong) [reply docs/internals/job_scheduling.zh.md]: Yes, I will merge you commit into Flink master codebase. 

Thanks for the reminder of it. I have converted it into a subtask of FLINK-11529.

**Pull request closed without merging**